### ISSUE TITLE
SR parks: add credit card field to forms

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1028]
@jlyon @aschmoe for some reason the credit card field is missing under pricing fields for San Rafael Parks.

Can you add that real quick?

**Comment by jlyon** [comment]
added @kevindherman did they fill out the sign up form already?

**Comment by kevindherman** [comment]
@jlyon they got this message when they submitted the sign up form:
![screenshot 2017-06-27 16 29 29](https://user-images.githubusercontent.com/6032426/27608537-eaac62d0-5b55-11e7-9af6-f8e73d88cbab.png)

**Comment by jlyon** [comment]
@kevindherman hmm, could we have them send a screenshot (if they feel comfortable), and we can work on fixing the issue and manually set up the acct?

**Comment by kevindherman** [comment]
@jlyon I'll ask. 

Where is the manual setup done from?

**Comment by jlyon** [comment]
@kevindherman I fixed the sign up form. Sorry, I hadn't tested it after all of the other Stripe dashboard updates.  Rebecca should be able to fill out the form at https://my.proudcity.com/sites/589046176d53977d90792ca5/payments/settings .  I can also submit the form for her if she can send the last 4 of her SSN (they were blanked out in Slack).

I set up livelifelocally with a test account, so they can build and test the form. See https://www.livelifelocally.org/wp-admin/admin.php?page=gf_edit_forms&id=4.  Transactions will show up in Forms, but not in the PC Dashboard.  After they have filled out the form (and once they tell us they are done testing). I will change the keys to their live keys and data will start to appear in the dashboard.
